contracts/hardhat.config.ts

```diff
@@ -1,9 +1,9 @@
-import '@typechain/hardhat'
 import '@nomiclabs/hardhat-ethers'
 import '@nomiclabs/hardhat-etherscan'
 import '@nomiclabs/hardhat-waffle'
-import 'hardhat-contract-sizer'
+import '@typechain/hardhat'
```

Comment:
Why these changes?

Reply:
I think these were just import ordering changes by the formatter to order them alphabetically.

Reply:
Agreed!